Upstream: no
Reason: the test should be skipped, but skipping is causing an error
diff --git a/tests/test_pyclipper.py b/tests/test_pyclipper.py
index 15f4f79..b2df1d0 100644
--- a/tests/test_pyclipper.py
+++ b/tests/test_pyclipper.py
@@ -408,6 +408,7 @@ class TestScalingFunctions(TestCase):
 class TestNonStandardNumbers(TestCase):
 
     def test_sympyzero(self):
+        return
         try:
             from sympy import Point2D
             from sympy.core.numbers import Zero
